MapboxSearch/discover/src/androidTest/java/com/mapbox/search/discover/DiscoverIntegrationTest.kt

```diff
@@ -75,7 +74,6 @@ internal class DiscoverIntegrationTest {
 
         val url = requireNotNull(request.requestUrl)
         assertEqualsIgnoreCase("//search/v1/category/${query.canonicalName}", url.encodedPath)
-        assertEquals(TEST_ACCESS_TOKEN, url.queryParameter("access_token"))
```

Comment:
We still should check if token is passed to backend.

Reply:
Fixed